.prettierrc.js

```diff
@@ -0,0 +1,3 @@
+// Import the default config file and expose it in the project root.
+// Useful for editor integrations.
+module.exports = require( '@wordpress/scripts/config/.prettierrc.js' );
```

Comment:
We should document it and ensure that this file is always usable outside. I guess it will become the blessed way to integrated with IDEs. Unless we expose a function that proxies the default configs provided.

```js
const { getDefaultConfig } from '@wordpress/scripts';

module.exports = getDefaultConfig( 'prettier' );
```

It's probably unnecessary indirection though :)